src/guides/v2.3/extension-dev-guide/service-contracts/design-patterns.md

```diff
@@ -148,6 +148,49 @@ Management interfaces provide management functions that are not related to repos
    </tr>
 </table>
 
+### Metadata interfaces {#metadata-interfaces}
+
+Metadata interfaces provide methods for retive metadata, the interfaces are not related to repositories. For example:
+
+<table style="width:100%">
```

Comment:
Hi @DenisSaltanahmedov. Please, feel free to use Markdown tables instead of HTML tables. It's a preferable way.